src/pylhe/__init__.py

```diff
@@ -269,12 +270,16 @@ def read_lhe_init(filepath):
                             try:
                                 wg_id = w.attrib["id"]
                             except KeyError:
+                                # Instead of raising an error, we could just use the index
+                                # wg_id = index
```

Comment:
Is this something that is going to be addressed in this PR? If not, then this type of comment would be better served in a GitHub Issue where it can become actionalbe.